MediaBrowser.Model/Search/SearchHint.cs

```diff
@@ -1,8 +1,8 @@
-#nullable disable
-#pragma warning disable CS1591
+#pragma warning disable CS0618
```

Comment:
Why is this needed? `ItemId` is referenced here, is it?

Reply:
Id and ItemId are the same, from SearchController:

```
            // legacy
            result.ItemId = result.Id;
```

So I want to mark ItemId as obsolete so it generates the `deprecated: true` field in the OpenAPI specification. Can't figure out how to disable the error during build though..

Reply:
I would expect the warning to be inside the `SearchController` file :thinking:

Reply:
Yeah that's why it didn't work